docs/static/diffs/expo-updates-android.diff

```diff
@@ -0,0 +1,102 @@
+diff --git a/android/app/build.gradle b/android/app/build.gradle
+index e5b1a7d..d06dfd0 100644
+--- a/android/app/build.gradle
++++ b/android/app/build.gradle
+@@ -81,10 +81,11 @@ import com.android.build.OutputFile
+ project.ext.react = [
+     enableHermes: false,  // clean and rebuild if changing
+ ]
+ 
+ apply from: "../../node_modules/react-native/react.gradle"
++apply from: "../../node_modules/expo-updates/scripts/create-manifest-android.gradle"
+ 
+ /**
+  * Set this to true to create two separate APKs instead of one:
+  *   - An APK that only works on ARM devices
+  *   - An APK that only works on x86 devices
+diff --git a/android/app/src/main/AndroidManifest.xml b/android/app/src/main/AndroidManifest.xml
+index 955757b..3fa4cb1 100644
+--- a/android/app/src/main/AndroidManifest.xml
++++ b/android/app/src/main/AndroidManifest.xml
+@@ -8,12 +8,15 @@
+       android:label="@string/app_name"
+       android:icon="@mipmap/ic_launcher"
+       android:roundIcon="@mipmap/ic_launcher_round"
+       android:allowBackup="false"
+       android:theme="@style/AppTheme">
+-      <activity
+-        android:name=".MainActivity"
++      <meta-data android:name="expo.modules.updates.EXPO_UPDATE_URL" android:value="https://exp.host/@my-expo-username/my-app" />
++      <meta-data android:name="expo.modules.updates.EXPO_SDK_VERSION" android:value="38.0.0" />
++
++  <activity
++    android:name=".MainActivity"
```

Comment:
this diff is a little weird, only the lines on 29 and 30 need to be added, the other changes in the diff are not meaningful

Reply:
good point, manually changed this